src/main/java/com/example/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.TechItEasy.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class TelevisionController {
+
+    @GetMapping("/televisions")
+    public ResponseEntity<Object> getAllTelevisions(){
```

Comment:
Ook bij je mappings mag je de Generic van de returntype van de methode zo precies mogelijk definieren. Dus ipv Object, kun je hier String typen. In je andere controller kun je Television gebruiken en bij mappings die geen body mee terug geven, kun je Void gebruiken.